presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -137,9 +137,9 @@ public void testTableWithNoSupportedColumns()
             assertQueryFails("SELECT * FROM no_columns", "Table 'tpch.no_columns' not found");
             assertQueryFails("SELECT 'a' FROM no_columns", "Table 'tpch.no_columns' not found");
 
-            assertQueryFails("SELECT c FROM non_existent", ".* Table .*tpch.non_existent.* does not exist");
-            assertQueryFails("SELECT * FROM non_existent", ".* Table .*tpch.non_existent.* does not exist");
-            assertQueryFails("SELECT 'a' FROM non_existent", ".* Table .*tpch.non_existent.* does not exist");
+            assertQueryFails("SELECT c FROM non_existent", ".*Table .*tpch.non_existent.* does not exist");
```

Comment:
why did the error regex change?

Reply:
With preprocessing enabled(its disabled by default at the moment), the error message also have the exception class name, hence adding regex.